src/vuex/modules/collection/users-actions.js

```diff
@@ -0,0 +1,34 @@
+import {
+  RECEIVE_DOCUMENTS,
+  DELETE_DOCUMENT
+} from './mutation-types'
+
+import kuzzle from '../../../services/kuzzle'
+
+export const deleteUser = (store, user) => {
+  if (!user.id) {
+    return
+  }
+
+  kuzzle
+    .security
+    .deleteUser(user.id, error => {
+      if (error) {
+        return
+      }
+
+      store.dispatch(DELETE_DOCUMENT, user.id)
+    })
+}
+
+export const searchUsers = (store, filters = {}) => {
+  kuzzle
+    .security
+    .searchUsers(filters, {hydrate: true}, (error, result) => {
```

Comment:
question: I don't know what's the kuzzle target branch for this BO. In case it is the `develop` branch then this option can be removed.